### ISSUE TITLE
Ceck if bounding boxes are valid in the data generator / training data 

### ISSUE THREAD

**Issue by cells2numbers** [broadinstitute/keras-rcnn#133]
When the training data contains invalid bounding boxes with ```box["x1"] > box["x2"]``` the loss drops to ```nan```. To solve this a simple check for box["x1"] < box["x2"] and box["y1"] < box["y2"] should be implemented.
